data_training.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2f201f51-5ade-40c6-8bf4-9b027b05eefb",
+   "metadata": {},
+   "source": [
+    "# Data Training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "335d12cd-96b4-454a-a84b-110a03ef24d5",
+   "metadata": {},
+   "source": [
+    "## Library Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5bf386b0-e55b-4063-afbd-ea1a8a09da27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "import pandas as pd\n",
+    "from sklearn.preprocessing import StandardScaler, LabelEncoder, OneHotEncoder\n",
+    "from sklearn.compose import make_column_transformer\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "import numpy as np"
+   ]
+  },
```

Comment:
- The imports are clear and well-organised.
- `make_column_transformer` from `sklearn.compose` is imported but not used. Consider removing it if it's not needed.

Reply:
Good catch.